contrib/raftexample/raft.go

```diff
@@ -247,7 +302,79 @@ func (rc *raftNode) stopHTTP() {
 	<-rc.httpdonec
 }
 
+func (rc *raftNode) applySnapshot(snapshotToSave raftpb.Snapshot) {
```

Comment:
We need to set `rc.confState` and others.

And kvstore that receives from `commitC` does not touch `rc.confState`, so we need these update parts here?